crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -43,17 +43,55 @@ pub struct Mesh {
 /// [`shape::Cube`](crate::mesh::shape::Cube) or [`shape::Box`](crate::mesh::shape::Box), but you can also construct
 /// one yourself.
 ///
-/// Example of constructing a mesh:
+/// Example of constructing a mesh (to be rendered with a StandardMaterial):
```

Comment:
```suggestion
/// Example of constructing a mesh (to be rendered with a `StandardMaterial`):
```
Or even a link if it would work here